src/test/java/org/mockitousage/internal/junit/UnusedStubbingsFinderTest.java

```diff
@@ -148,4 +150,37 @@ public void deduplicates_stubbings_by_location() throws Exception {
         // then
         assertEquals(1, stubbings.size());
     }
+
+    @Test
+    public void bdd_unused_and_lenient_stubbings() throws Exception {
+        given(mock1.simpleMethod(1)).willReturn("1");
+        given(mock1.simpleMethod(2)).willReturn("2");
+        lenientBDD().given(mock2.simpleMethod(3)).willReturn("3");
```

Comment:
Can you come up with a strategy so that we can write:

```
lenient().given(...)...
```

We'd like to understand what are options and trade-offs.